### PR TITLE
Issue #92 - New /flag command.

### PR DESCRIPTION
**Pull request by PureKrome**
- Added new /flag command.
- Added unit tests to test the new command.
- Added DB migration for a new DB field.

### CODE REVIEW

File: JabbR/Services/ChatService.cs [hunk 1]
```diff
@@ -490,5 +745,24 @@ internal static void ValidateNote(string note)
...
+            if (String.IsNullOrWhiteSpace(country))
+            {
+                throw new InvalidOperationException(
+                    "Sorry, but the country ISO code you requested doesn't exist. Please refer to http://en.wikipedia.org/wiki/ISO_3166-1_alpha-2 for a proper list of country ISO codes.");
+            }
+        }
+
+        internal static string GetCountry(string isoCode)
+        {
+            return String.IsNullOrWhiteSpace(isoCode) ||
```
Comment (davidfowl) [inline JabbR/Services/ChatService.cs]: Too much stuff in here, doesn't need to be this complicated. 

```
if(String.IsNullOrEmpty(isoCode)) {
    return null;
}

string country;
if(Countries.TryGetValue(isoCode, out country)) 
{
       return country;
}

return null;
```

File: JabbR/Commands/CommandManager.cs [hunk 2]
```diff
@@ -809,5 +815,26 @@ private void HandleAfk(ChatUser user, string[] parts)
 
             _repository.CommitChanges();
         }
+
+        private void HandleFlag(ChatUser user, string[] parts)
+        {
+            string isoCode = String.Join(" ", parts.Skip(1)).Trim().ToLowerInvariant();
```
Comment (davidfowl) [inline JabbR/Commands/CommandManager.cs]: Don't do this. Just take parts[1] and do ToLowerInvariant

Comment (PureKrome) [inline JabbR/Commands/CommandManager.cs]: NP :)

File: JabbR/Services/ChatService.cs (hunk 1 above)
Comment (PureKrome) [inline JabbR/Services/ChatService.cs]: Also NP :)

**Pull request merged**